pkg/utils/cmd/tiller-hack/tiller-hack.go

```diff
@@ -0,0 +1,182 @@
+package main
+
+import (
+	"bytes"
+	"compress/gzip"
+	"encoding/base64"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"os"
+
+	"github.com/sirupsen/logrus"
+
+	// "gopkg.in/yaml.v2"
+	// "github.com/ghodss/yaml"
+	"k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
+	"k8s.io/client-go/util/retry"
+
+	"github.com/golang/protobuf/proto"
+	"k8s.io/helm/pkg/proto/hapi/chart"
+	rspb "k8s.io/helm/pkg/proto/hapi/release"
+)
+
+var (
```

Comment:
re kubeconfig - can't you just use istio/pkg/kube/config.go for this functionality?

Reply:
well, if I read that code right, it would require more than one call to get Clientset. I do not see value of replacing one cal I have with multiple to get the same thing.